### DISCUSSION TITLE
🇧🇷 Introducing myself 👋🏼

### DISCUSSION THREAD

**Discussion by vanesagab** [community/community#20059] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi everyone! I'm Vane from Brazil and just joined this awesome community! 

Nice to meet you all.

**Comment by radar-blip** [comment]
Greetings from USA.

**Comment by kaiopiola** [comment]
Greetings, Vane! Nice to meet you!

I'm Kaio, also from Brazil! I hope to see your projects succeeding here!

**Comment by manasesjesus** [comment, marked as answer]
Oi menina :) Welcome to GH! You may consider creating a profile readme to introduce yourself. It's fairly simple and there are different ways to accomplish it. To start, here's GitHub's step-by-step [profile README tutorial](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme).

May the code be with you!

**Reply by vanesagab** [reply]
Wow, nice! I will definetley make mine. 
Obrigada!
